packages/bundler-plugin-core/src/options-mapping.ts

```diff
@@ -52,57 +57,84 @@ export type InternalIncludeEntry = RequiredInternalIncludeEntry &
   };
 
 export function normalizeUserOptions(userOptions: UserOptions): InternalOptions {
-  let entries: (string | RegExp)[] | ((filePath: string) => boolean) | undefined;
-  if (userOptions.entries === undefined) {
-    entries = undefined;
-  } else if (typeof userOptions.entries === "function" || Array.isArray(userOptions.entries)) {
-    entries = userOptions.entries;
+  const entries = normalizeEntries(userOptions.entries);
+  const include = normalizeInclude(userOptions);
+
+  return {
+    // Strictly required options
+    include,
+
+    // These options must be set b/c we need them for release injection
+    // They can also be set as environment variables. Technically, they
+    // could be set in the config file but this would be too late for
+    // release injection
+    org: userOptions.org ?? process.env["SENTRY_ORG"],
```

Comment:
H: Are we sure that `??` works with all versions of Node someone might be using? Is there a reason not to use `||` here, given that there are no valid falsy values possible here?

(I see that later on we're using `??` for booleans, where clearly `||` won't cut it, but it'd be simple to polyfill with something like 
```js
const useOptionOrDefault /* or choose some better name */ = (
  optionValue,
  defaultValue
) =>
  optionValue !== undefined && optionValue !== null
    ? optionValue
    : defaultValue;
```
.)

Reply:
Good catch, thanks! Node 10 (which we're targeting as the min. supported version) [doesn't support](https://node.green/#ES2020-features--nullish-coalescing-operator-----) the nullish coalescing operator. 

It's also a behaviour change for the string values but I think we can justify falling back to  the env/default values if users provided `""`, WDYT? Regardless, I'm wondering if we simply should use the `useOptionOrDefault` option in our source code or polyfill it while transpiling. Our build setup uses rollup with Babel for transpilation. There surely is a way to do this but I have to look into it yet. Any preferences?

Reply:
Okay, I should have taken a look at our output: We already transpile the operator away. Here's how the output for the `orgs` option above looks:

```
org: (_userOptions$org = userOptions.org) !== null && _userOptions$org !== void 0 ? _userOptions$org : process.env["SENTRY_ORG"],
```

Reply:
Ah, right. Didn't know how low you were targeting your down-compiling.

Seeing that output is a good reminder, though, that we should use `||` where we can for anything that'll be used in the browser - way fewer bytes than the polyfill if we don't actually need it.